### ISSUE TITLE
Themes Menu

### ISSUE THREAD

**Issue by re-LIF3** [Studio-42/elFinder#3679]
How to add Menu Themes at Preference ? theres is no document for it ? 
![Screenshot 2024-10-11 185813](https://github.com/user-attachments/assets/ad65f422-5dc7-4e1b-ba68-23eb5efecb4a)

**Comment by blutorange** [comment]
Yeah you kinda have to read some source code: https://github.com/Studio-42/elFinder/blob/master/js/commands/preference.js#L23-L32

E.g. to only show the `Theme` tab:

```js
$(".elfinder").elfinder({
  commandsOptions: {
    preference: {
      categories: { theme: ["theme"] },
    },
  },
})
```

And make sure you actually have some themes defined, [or I believe it won't show up either](https://github.com/Studio-42/elFinder/blob/master/js/commands/preference.js#L106):

https://github.com/Studio-42/elFinder/blob/master/js/elFinder.options.js#L235-L254

```js
{
            themes : {
                'dark-slim'   : 'https://johnfort.github.io/elFinder.themes/dark-slim/manifest.json',
                'mt-darkblue' : 'https://robinn1.github.io/elFinder-Material-Theme/manifests/darkblue.json',
                'mt-gray'     : 'https://robinn1.github.io/elFinder-Material-Theme/manifests/gray.json',
                'mt-light'    : 'https://robinn1.github.io/elFinder-Material-Theme/manifests/light.json',
                'bootstrap'   : 'https://nao-pon.github.io/elfinder-theme-manifests/bootstrap.json',
                'moono'       : 'https://nao-pon.github.io/elfinder-theme-manifests/moono.json',
                'win10'       : 'https://nao-pon.github.io/elfinder-theme-manifests/win10.json'
            },
}
```

**Comment by tomekboy** [comment]
hi team,

i would love to offer my users the option to change the elfinder theme. BUT can't get it to work...

i'm running elfinder just from the default elfinder.html and so far i set the desired options in the.html java script part of elfinder.html! the only file occurence (as from your descriptions / wicki, help) of getting themes into my elfinder is the elfinder.full.js. i tried to set the option / options:

`	themeid : { 'win10': 'https://nao-pon.github.io/elfinder-theme-manifests/win10.json'},
    
	themes : {
	'dark-slim'   : 'https://johnfort.github.io/elFinder.themes/dark-slim/manifest.json',
        'mt-darkblue' : 'https://robinn1.github.io/elFinder-Material-Theme/manifests/darkblue.json',
        'mt-gray'     : 'https://robinn1.github.io/elFinder-Material-Theme/manifests/gray.json',
        'mt-light'    : 'https://robinn1.github.io/elFinder-Material-Theme/manifests/light.json',
        'bootstrap'   : 'https://nao-pon.github.io/elfinder-theme-manifests/bootstrap.json',
        'moono'       : 'https://nao-pon.github.io/elfinder-theme-manifests/moono.json',
        'win10'       : 'https://nao-pon.github.io/elfinder-theme-manifests/win10.json'
},

	/**
	 * Theme id to initial theme
	 * 
	 * @type String|Null
	 */
	theme : 'win10',`

but whatever i tried (changing syntax) elfinder just shows the default theme.

and the preference option set in the elfinder.html:
`						preference: {
							categories: { theme: ["Darstellung (UI)"] },
						},`

shows no tab and an empty "body", meaning no choices are popping up like in or demo site!

any hints, tipps are highly appreciated!
cheers
tomek

**Comment by tomekboy** [comment]
good morning team,

i did find out how to set a theme by changing the defopts: in elfinder.html. BUT i am stll lacking the feature to offer the themes being chosen by the user through preferences. how can i achieve that?

thanks in advance for your kind consideration to help...

cheers
tomek

**Comment by tomekboy** [comment]
hi team,

all set! solved...
 i was able to set the options in the elfinder.html file accordingly to get my desired UI! here is what i did:
1. adding the themes
`					themes: {
						'dark-slim'   : 'https://johnfort.github.io/elFinder.themes/dark-slim/manifest.json',
						'mt-darkblue' : 'https://robinn1.github.io/elFinder-Material-Theme/manifests/darkblue.json',
						'mt-gray'     : 'https://robinn1.github.io/elFinder-Material-Theme/manifests/gray.json',
						'mt-light'    : 'https://robinn1.github.io/elFinder-Material-Theme/manifests/light.json',
						'bootstrap'   : 'https://nao-pon.github.io/elfinder-theme-manifests/bootstrap.json',
						'moono'       : 'https://nao-pon.github.io/elfinder-theme-manifests/moono.json',
						'win10'       : 'https://nao-pon.github.io/elfinder-theme-manifests/win10.json'
					},`
2. setting the preference options
`						preference: {
							categories: { theme: ["theme"] }
						},`

maybe this help help others?!

cheers & thank you for your wonderful elFinder!!!!!
tomek
